### PR TITLE
Cluck graphy graphs

### PR DESCRIPTION
**Pull request by astro-nat**

### CODE REVIEW

File: app/assets/stylesheets/default.scss [hunk 1]
```diff
@@ -45,4 +45,68 @@ br {
...
+
+// .btn-secondary:not(:disabled):not(.disabled).active,.btn-secondary:not(:disabled):not(.disabled):active,.show>.btn-secondary.dropdown-toggle {
+//     color:#fff;
+//     background-color:#545b62;
+//     border-color:#4e555b;
+// }
+
+// .btn-secondary:not(:disabled):not(.disabled).active:focus,.btn-secondary:not(:disabled):not(.disabled):active:focus,.show>.btn-secondary.dropdown-toggle:focus {
+//     box-shadow:0 0 0 .2rem rgba(108,117,125,.5);
+// }
```
Comment (constantinahug) [inline app/assets/stylesheets/default.scss]: Is all of this stuff commented out? Why is it included?

File: app/controllers/homepage_controller.rb [hunk 2]
```diff
@@ -23,36 +24,28 @@ def dashboard
       goal_values = Array.new
       current_effective_user.workout.each do | workout |
         workout.task.each do |task|
+          puts "G unit name: "
+          puts g.unit.name
           if task.exercise_id == g.exercise_id then
             task.exercise_set.each do | exercise_set |
-              goal_values.push(Array.new([exercise_set.created_at.strftime("%D %H:%M"), exercise_set.rep_value]))
+              if exercise_set.rep_unit == g.unit.name then
```
Comment (constantinahug) [inline app/controllers/homepage_controller.rb]: What does this mean?

File: app/assets/stylesheets/default.scss (hunk 1 above)
Comment (astro-nat) [reply app/assets/stylesheets/default.scss]: I was including some bootstrap source code to fix buttons but didn't use it, but also don't want to lose it.

File: app/controllers/homepage_controller.rb (hunk 2 above)
Comment (astro-nat) [reply app/controllers/homepage_controller.rb]: This section populates data to add to the dashboard graphs, so this prevents exercise logs from being added that don't apply for the goal units. So for now if a goal is Run "60 minutes" for example and someone logs Run "60 seconds" for an exercise, it doesn't populate the graph.

Comment (constantinahug) [reply app/controllers/homepage_controller.rb]: Sweet, thanks for the clarity!

File: app/assets/stylesheets/default.scss (hunk 1 above)
Comment (constantinahug) [reply app/assets/stylesheets/default.scss]: Cool, makes sense

**Pull request merged**